Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt

```diff
@@ -5,7 +5,7 @@
 // Description: looks for anomalous number of resources creation or deployment activities in azure activity log.
 // It is best to run this query on a look back period which is at least 7 days.
 //
-// DataSource: #AzureActivity
+// DataSource: #AzureActivity, #Impact
```

Comment:
ditto - this change should be in Tactics line?

Reply:
fixed